projects/1.21/assets/occultism/occultism/lang/zh_cn.json

```
@@ -0,0 +1,3191 @@
+  "book.occultism.dictionary_of_spirits.pentacles.light_blue_chalk.lore.title": "万年不化",
+  "book.occultism.dictionary_of_spirits.pentacles.light_blue_chalk.name": "冰川粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.light_blue_chalk.recipe_impure.text": "魔精无法在不进行烧炼的前提下粉碎冰，你需要更高等级的熔炼者。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.light_blue_chalk.spotlight.text": "尽管主要由冰制成，但其中的魔法力量使其不会融化。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.light_gray_chalk.description": "淡灰色粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.light_gray_chalk.lore.text": "淡灰色粉笔通过矿物化合物强化五芒星阵的阵基。\n阵基是重要属性，常被视为“核心”的稳定载体。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.light_gray_chalk.lore.title": "最常用的阵基",
+  "book.occultism.dictionary_of_spirits.pentacles.light_gray_chalk.name": "优良阵基粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.light_gray_chalk.spotlight.text": "由于成本相对较低，且具有天然的魔法特性，这种材料成为神秘学研习者最常用的阵基选择，几乎能在所有五芒星阵中替代白色粉笔。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.lime_chalk.description": "黄绿色粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.lime_chalk.lore.text": "黄绿色粉笔比魔精更能吸引强大的魔灵。\n任何希望提升仪式等级的人都需要这种粉笔。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.lime_chalk.lore.title": "获取经验",
+  "book.occultism.dictionary_of_spirits.pentacles.lime_chalk.name": "知识粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.lime_chalk.spotlight.text": "由灌注经验的珍稀宝石制成，这些黄绿色符文因此意义非凡，彰显你的技艺已突破初级阶段。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.magenta_chalk.description": "品红色粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.magenta_chalk.lore.text": "龙之伟力尽凝于一笔：此乃紫色粉笔之神迹。\n魔灵与龙的关系尚不明确，但必定有所关联。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.magenta_chalk.lore.title": "末影能量",
+  "book.occultism.dictionary_of_spirits.pentacles.magenta_chalk.name": "龙之粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.magenta_chalk.spotlight.text": "啊，末路之地的能量。但究竟是怎样的末路？末路当真存在吗？\n在末路处究竟了结了什么？末路本身可会有终结之时？\n",
+  "book.occultism.dictionary_of_spirits.pentacles.name": "五芒星阵",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.description": "橙色粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.honey.text": "要获得用来制作[](item://occultism:chalk_orange)的[](item://occultism:cursed_honey)，你需要[召唤并击杀一只被附身的蜜蜂](entry://possession_rituals/possess_bee)。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.lore.text": "橙色粉笔对火灵级别的魔灵而言是绝佳诱饵，虽然它们也会被黄绿色粉笔吸引，却能抵抗其召唤。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.lore.title": "火灵引召",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.name": "芬香粉笔",
+  "book.occultism.dictionary_of_spirits.pentacles.orange_chalk.spotlight.text": "作为一块甜美而略带温热的粉笔，它并不能保证能够掌控召唤出的火灵。\n要驱使火灵这类魔灵，需要与其魔灵类型直接相关的力量，以及五芒星阵中极致的稳定性。\n",
+  "book.occultism.dictionary_of_spirits.pentacles.paraphernalia.candle.text": "蜡烛能够提升五芒星阵的稳定性，从而减缓所召唤魔灵的灵髓衰退，使魔灵、被附身的目标或存在得以延长存续时间。\n",
```

```suggestion
  "book.occultism.dictionary_of_spirits.pentacles.paraphernalia.candle.text": "蜡烛能够提升五芒星阵的稳定性，从而减缓所召唤魔灵的灵髓衰退，使魔灵、被附身的物体或存在得以延长存续时间。\n",
```
应该是物体……吧，不过改不改应该问题不大